kubeflow_clusters/issue-label-bot/gcp_config/cluster-kubeflow.yaml

```diff
@@ -4,7 +4,7 @@ resources:
 - name: kubeflow
   properties:
     autoprovisioning-config:
-      enabled: true
+      enabled: false
```

Comment:
this removes autoprovisioning config from deployment manager config, but when applying the config to GCP, it will not disable autoprovisioning.
Therefore, I need to disable autoprovisioning manually again.